civihr_employee_portal/views/templates/views-view--default.tpl.php

```diff
@@ -27,6 +27,31 @@
  * @ingroup views_templates
  */
 ?>
+
+<?php
+
+  // Markup generated from views was not integrating well with panels in the
+  // current styling. For views in $views_with_generic_display_output array
+  // is being provided a different template "_views-view--generic-display-output.tpl.php"
+  // to get rid of specifc views HTML structure and classes to take advantage
+  // of existing styles imitating panels blocks structure.
+
+  // Logic to change template for specific views
+  // without increasing the number of files used
+  // templates
+  $views_with_generic_display_output = [
+    'MyDetails_MyAddress',
+    'MyDetails_PayrollInformation',
+    'MyDetails_Personal',
+    'Emergency Contacts',
+  ];
+
+  if ( in_array($view->get_human_name(), $views_with_generic_display_output) ) {
```

Comment:
@powdevel can you move to preprocess function please.

Reply:
@hitesh-jain Sure. Thanks.